build/common.props

```diff
@@ -1,16 +1,16 @@
 <Project>
   <Import Project="..\version.props" />
-  <Import Project="..\.build\common.props" Condition="Exists('..\.build\common.props')" />
+
+  <PropertyGroup>
+    <!-- these imports substitute for a package reference to Internal.AspNetCore.Sdk -->
+    <CustomBeforeMicrosoftCommonProps>$(MSBuildThisFileDirectory)..\src\Internal.AspNetCore.Sdk\build\Internal.AspNetCore.Sdk.props</CustomBeforeMicrosoftCommonProps>
```

Comment:
`<_SdkProjectDirectory>$(MSBuildThisFileDirectory)..\src\Internal.AspNetCore.Sdk</_SdkProjectDirectory>` and reuse. Probably better if we do

`$([MSBuild]::GetDirectoryNameOfFileAbove($(MsBuildThisFileDirectory), NuGet.config))` since it makes multiple levels of folder hierarchy possible without hardcoding the relative directory structure